Towny/src/main/java/com/palmergames/bukkit/towny/object/jail/Jail.java

```diff
@@ -102,13 +102,12 @@ public void addJailCell(Position position) {
 		TownyUniverse.getInstance().addSpawnPoint(new SpawnPoint(position, SpawnPointType.JAIL_SPAWN));
 	}
 	
-	public void removeJailCell(Location loc) {

```

Comment:
This method doesn't have to get removed and if it was removed it has be marked deprecated for a period of time anyways because it is `public`.